packages/scheduler/src/__tests__/SchedulerPostTask-test.js

```diff
@@ -178,90 +189,115 @@ describe('SchedulerPostTask', () => {
     scheduleCallback(NormalPriority, () => {
       runtime.log('B');
     });
-    runtime.assertLog(['Post Task']);
-    runtime.fireNextTask();
-    runtime.assertLog(['Task Event', 'A', 'B']);
-  });
-
-  it('multiple tasks with a yield in between', () => {
```

Comment:
Where did this test go?

Reply:
Doesn’t really make sense anymore, since we don’t control the task queue in this implementation. To test the same behavior, we’d need to write an e2e test.